src/integrationTest/java/uk/gov/hmcts/cmc/claimstore/RestTestClient.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.cmc.claimstore;
+
+import io.restassured.RestAssured;
+import io.restassured.response.Response;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.claimstore.processors.JsonMapper;
+import uk.gov.hmcts.cmc.domain.models.ClaimData;
+
+@Component
+public class RestTestClient {
+
+    private final TestsSetup testsSetup;
+    private final JsonMapper jsonMapper;
+
+    @Autowired
+    public RestTestClient(
+        TestsSetup testsSetup,
+        JsonMapper jsonMapper
+    ) {
+        this.testsSetup = testsSetup;
+        this.jsonMapper = jsonMapper;
+    }
+
+    public Response post(ClaimData claimData) {
+        return RestAssured
```

Comment:
I like this formatting, but what happens when you press auto-format short cut?

Reply:
It will align them to one line ☹️  I'll align it in the same way to avoid unnecessary whitespace changes.